data/os/AlmaLinux.9.yaml

```diff
@@ -0,0 +1 @@
+puppetboard::python_version: '3.9'
```

Comment:
do we still have to set this? ugh :(

if required, please add the `---` as first line to have it consistent.